charts/reaper-operator/values.yaml

```
@@ -13,8 +13,9 @@ commonLabels: {}
 replicaCount: 1
 
 image:
+  registry: docker.io
```

Missing docs comment

```suggestion
  # -- Container registry containing the repository where the image resides.
  registry: docker.io
```

---

@jsanda can we get this resolved and rerun the helm docs script?